README.md

```diff
@@ -124,29 +127,47 @@ class MyComponent {
 }
 ```
 
-The HTTP will include the following Connection types, and will provide an interface to ensure third-
-party Connection implementations conform to the expected API.
+The Http library will include the following `Connection` factories, and will provide an interface to
+ensure third-party `Connection` implementations conform to the expected API.
 
- * XHRConnection
- * JSONPConnection
- * NodeConnection
- * MockConnection
+ * Http
```

Comment:
How do we deal with things like fxos's weird non-standard XHR constructor? Just tell people to create a new Connection class to build it?